internal/series/series.go

```diff
@@ -52,7 +52,7 @@ var Series = map[string]SeriesInfo{
 	"yakkety": {true, Ubuntu, true, 1.103},
 	"zesty":   {true, Ubuntu, true, 1.104},
 	"artful":  {true, Ubuntu, true, 1.105},
-	"bionic":  {true, Ubuntu, true, 0}, // TODO 1.15 when released
+	"bionic":  {true, Ubuntu, true, 1.15},
```

Comment:
shouldn't this be 1.5 or something higher than 1.375?

Reply:
I think this looks right, 1.15 is higher than xenial's 1.1375.

Reply:
My bad, I read that as 1.375 and missed the 1 in there. I retract my comment.